### ISSUE TITLE
Make the normal body font on HE 25% darker.

### ISSUE THREAD

**Issue by lostboyevsky** [jetlej/HE-6.0-Bugs#282]
@JonBrooksX @MSchirp, do you agree with this?

I've thought for a while that the body font is too light, and we just got an email from someone saying to save his eyes and make our font "bolder." I assume he meant darker.

**Comment by MSchirp** [comment]
This mail?

```
undefined: 7968fdeb45

undefined: /contact/

Name: william

Email: klingler3@gmail.com
Subject: text/font??

Message:
We used to print in black ink on white paper to save ink. Now to save battery life and my eyes, you should have white lettering on a black background. And maybe pick a bolder font for your articles. Thx
```

If so, ignore it.

**Comment by JonBrooksX** [comment]
Even if that comment is bogus, I still think we can improve the typography. 

I suggest...

*For body:*

Font family: Iowan
Color: #1b1b1b

*For blockquotes:*

Font family: Fira Sans
Color: #1b1b1b

----

I took some screenshots of what that might look like. 

![screenshot 2016-04-08 21 37 54](https://cloud.githubusercontent.com/assets/13107838/14396939/6e3cd9c8-fdd2-11e5-98e5-92230e26b211.png)
![screenshot 2016-04-08 21 37 38](https://cloud.githubusercontent.com/assets/13107838/14396938/6e3bb6c4-fdd2-11e5-8aa9-83d90daf25f2.png)
![screenshot 2016-04-08 21 31 49](https://cloud.githubusercontent.com/assets/13107838/14396940/6e40132c-fdd2-11e5-938a-006fb4da2c35.png)
![screenshot 2016-04-08 21 31 37](https://cloud.githubusercontent.com/assets/13107838/14396941/6e488f8e-fdd2-11e5-8de8-6a1f90d47def.png)
![screenshot 2016-04-08 21 30 58](https://cloud.githubusercontent.com/assets/13107838/14396942/6e4bdd4c-fdd2-11e5-92a1-23a65bdaff50.png)

**Comment by JonBrooksX** [comment]
The thing that doesn't look good is the bolded bits of quotes, but that's easily fixable.

**Comment by jetlej** [comment]
I like that new font :) See if you can find a medium boldness for bold words though.

**Comment by JonBrooksX** [comment]
Okay, great, will do.

**Comment by lostboyevsky** [comment]
@JonBrooksX, my thoughts:

i don't think Iowan is superior to our current main body font. i would rather stick with our current font but simply make the normal text (not the bolded text) about ~15-20% darker. 

i think the Fira Sans is far superior to our current blockquote font and would like to implement it immediately. the only problem with it is the bolded text. it's too black and also too thick. if there's a way to make it lighter and a bit thinner, i say we have a winner and should implement asap.

**Comment by MSchirp** [comment]
I trust your guys judgment on this. Please be very clear with instruction for Deepak once you've finalised your decision.

**Comment by JonBrooksX** [comment]
I don't agree with Iowan not being superior to our current font. Iowan is one of the best, oldest reading fonts out there and the default for iBooks. But I'm fine if you don't like it. Let's go with your suggestion for now since it's quick to implement. Let's keep our eyes open for other fonts though.

I'm going to think of this thread as the body-font thread, and use another quote thread for the quotes since I've already fleshed out a design on the other thread. 

----

*Instructions for @dpatil-gmcs:*

Change the current font-family (Lora) to the color: #1b1b1b

**Comment by lostboyevsky** [comment]
@JonBrooksX @dpatil-gmcs 

Upon further inspection/consideration, I agree with Jon that Iowan is a better body font than Lora. Therefore, I think we should change the body font to Iowan and the blockquote font to Fira Sans ASAP. 

@JonBrooksX, I'm not sure how you determined the specific best hex code color for the body font. Can you provide your ideal choice for the blockquote font color as well?

**Comment by JonBrooksX** [comment]
Okay, that's cool. Thanks for reconsidering. 

With the font color it's just the color that Brain Pickings uses. It's not black, but almost. Pure black is a bit too bold IMO. Our current font color is too light, so I figured #1b1b1b would be good to use. I think we should use the same color for our quotes too.

**Comment by lostboyevsky** [comment]
@JonBrooksX, does Brain Pickings use the same font color for normal body text and blockquotes?

@dpatil-gmcs, are these instructions clear enough?

**Comment by JonBrooksX** [comment]
Just checked @lostboyevsky 

Body font color: #262626

Block quote color: #1a1a1a

**Comment by MSchirp** [comment]
@JonBrooksX Please give @dpatil-gmcs clear instruction for all steps and Ill move this up in his priority list.

**Comment by lostboyevsky** [comment]
@dpatil-gmcs, here are the steps to take:

1. Change the HE body font to Iowan.

2. Change the HE blockquote font to Fira Sans. 

3. Change the body font color to #262626 

4. Change the blockquote font color to #1a1a1a

**Comment by dpatil-gmcs** [comment]
@lostboyevsky  @MSchirp  @JonBrooksX 

We have implemented the above task and deployed the code to the LIVE site,
Please have a look and let us know your thoughts on it,
Also give us a sign-off so that we can close the issue.
